### ISSUE TITLE
ASIAN MEMBERS ATTENTION: RAM meetings at Asian time

### ISSUE THREAD

**Issue by zsluedem** [rchain/bounties#834]
### Benefit to RChain

In #833 , Jonathanks( I don't know his github name) and I have talked about the time for regular RAM meeting is really hard for Asian members to participate. So we are trying to do a regular meeting at Asian time, maybe twice a month. 

The first meeting will hold at **04:00 UTC 20180717**
Sydney +11, Beijing +8, Greece +3, Amsterdam +2, Lagos +1,
New York -4, Seattle -7

The zoom link is **not confirmed** now.

- [ ] Prepare [agenda]().

@lapin7 @allancto @makys @raob @ian-bloom @Jake-Gillberg @Ojimadu @kitblake @Tonyprisca13
@pmoorman @AyAyRon-P @dckc @Mervyn853 

_**Do you guys know any active members from Asian like  India, Vietnam, and Australia, Japan and Korea?  Please tell them about this meeting.**_

### Budget and Objective

*Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely.*

_Estimated Budget of Task:_ **$[TBD]**
_Estimated Timeline Required to Complete the Task:_ **[days/weeks]**
_How will we measure completion?_ **[example: pull request merged]**

*See CONTRIBUTING.md for details on budget and reward process.*

**Comment by pmoorman** [comment]
excellent idea! 

Count me in as a RAAM from October/November onwards again. Btw, @zsluedem I think you're doing an excellent job bridging between Asia and the rest of the community. Great work!

**Comment by ottermagically** [comment]
Great conversation in Discord the past few hours, @zsluedem ... I am going to think more and _possibly_ refine our goals or create an additional issue.

**Comment by Viraculous** [comment]
@zsluedem and @ottermagically Is the RAAM going to be in English as well? What about members who are not may not have that English inclination? Is there a way to as well communicate with them?

**Comment by zsluedem** [comment]
The meeting is not only for Chinese members so it is going to be English as well.

**Comment by Viraculous** [comment]
@zsluedem great, I was only concerned localisation of the information for members not inclined to English language so that they can as well roll along with the community. How about Greg's debriefing, does it go well with the time zone in Asia?

**Comment by allancto** [comment]
@viraculous good questions, @zsluedem @ottermagically good answers! Making this RAAM meeting on par with the current RAM but offset 12 hours will make attending one or both convenient for all of our members around the world. (may i propose the name RAM+12 or similar?)

I'm particularly heartened by @zsluedem's remark about common language, whether English or another, because it facilitates Cooperation rather than fragmentation despite a few potential challenges. Perhaps it's off in the future, but I look forward to simultaneous interpretation/translation or whatever it takes to make it convenient for all of our members to be able to collaborate worldwide and in real time. 

Awesome @ottermagically @zsluedem. -@allancto

**Comment by zsluedem** [comment]
@Viraculous @allancto Actually @ottermagically  and I want people from  as many countries in Asia as possible so English is still the priority.That's the only way to communicate with each other well because there are all kinds of languages in Asia like Japanese, Chinese, Korean. I think there is no easy way to get rid of the language thing.

BTW, do you guys know any members from other countries in Asia like Japan ,Korea, India .etc?

**Comment by Mervyn853** [comment]
Count me in. Will be glad to jump on the weekly call whenever my schedule fits in. Would love to help grow the RChain Asian community.

**Comment by Viraculous** [comment]
@zsluedem says 

> @Viraculous @allancto Actually @ottermagically and I want people from as many countries in Asia as possible so English is still the priority.That's the only way to communicate with each other well because there are all kinds of languages in Asia like Japanese, Chinese, Korean. I think there is no easy way to get rid of the language thing.

> 
I agree with you @zsluedem but am looking at it from a point of scalibility. Great, English language is universal and largely spoken but more concerned about effective communication within the localities with less or no inclineation to English language. To start with, English language is ok but subsequently as the community and awareness about RChain grows, will it scale? If not, the need arises for a plan on the localisation of the information/conversations/hangouts to the language understood by the major locality. Its a step quick penetration into communities.

**Comment by dckc** [comment]
@justinc667 I hope you can get involved in this.

_I tried to assign you, but you don't seem to be a collaborator. Ask @lapin7 about that._

**Comment by zsluedem** [comment]
@dckc What is the handle of @justinc667 in discord? Or email?
Can you make a small introduction of him?

**Comment by allancto** [comment]
J could j

On Jul 13, 2018 9:51 PM, "will" <notifications@github.com> wrote:

@dckc <https://github.com/dckc> What is the handle of @justinc667
<https://github.com/justinc667> in discord? Or email?
Can you make a small introduction of him?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/rchain/bounties/issues/834#issuecomment-404998722>, or mute
the thread

**Comment by zsluedem** [comment]
@ottermagically @pmoorman @Mervyn853 @Viraculous @allancto @justinc667 @dckc 
I have created an agenda doc for the meeting. 
https://docs.google.com/document/d/1kH1RKMnM9jTkNy8T2AZyZqJj92i7D4dcjuu4Z0n_Yzk/edit?usp=sharing
Please add anything you want to discuss.

**Comment by pmoorman** [comment]
To support this great initiative I'll get out of bed at 6am (I'm in Amsterdam now).

**I'll volunteer my Zoom room** if it's of any help (https://zoom.us/j/3287944287). 

As for the agenda, I would be really curious to hear an update from @ottermagically about his good works in Singapore, and maybe from @zsluedem to cover the status in China. I'd also like to discuss if there's anything people like myself can do to be more welcoming / inviting to especially the Chinese community, and invite collaboration between Chinese and non-Chinese members. 

Those are just ideas, but maybe it helps in shaping the agenda.

_@kitblake is in Tokyo. Maybe he can attend, too._

**Comment by allancto** [comment]
@ottermagically did i understand correctly that @traviagio is currently based in Singapore and may be interested? also the contributors to #776, including @ddayan and @jeremybeal11, following Cardano and asian based blockchains, would RAAM be of interest to you? @mrinalmanohar are you all developing contacts near this time zone, would you be interested in this meeting (midnight ny iiuc)?

**Comment by zsluedem** [comment]
The zoom link is updated. See you guys next week.

**Comment by ddayan** [comment]
5am London time... I might have to catch up with some members after the meeting.

**Comment by ottermagically** [comment]
PLEASE NOTE A NEW ZOOM URL SO THAT SESSION MAY BE MORE EASILY RECORDED AND SAVED TO YOUTUBE:

https://zoom.us/j/2522161381

**Comment by justinc667** [comment]
@zsluedem  My tags online are KB667 {Justin}#8495 on discord which is short for kaboss667 elsewhere. I spent a semester abroad in Japan during my 4-year degree in Japanese Literature and Language. I will be in the meeting looking to branch out to communities abroach, specifically in Japan.

**Comment by allancto** [comment]
@ottermagically please accept your github invitation to collaborate, then you will have edit privileges. Any questions just DM me. Thanks!! -@allancto

**Comment by Viraculous** [comment]
Great @Justinc667 @zluedem @ottermagically

**Comment by zsluedem** [comment]
I am so glad we did it ~! I will write a summary about the meeting today from the doc and report it in the next RAM.

**Comment by Viraculous** [comment]
Great @zsluedem

**Comment by zsluedem** [comment]
Hello every one. We start a good meeting every week right now. @pmoorman and I suggest 3000$ budgets for the meeting this month. The budget reward includes #849 #861 .

In the next month , we are going proposed 1500$ each month on the Rchain Asia Weekly Debrief .

**Comment by Ojimadu** [comment]
@pmoorman @zsluedem What is the idea behind the $3000? Being that this is a subset of the regular RAM meeting and @deannald is [suggesting a budget](https://github.com/rchain/bounties/issues/823#issuecomment-405065113) of $100-$150 for the RAM meeting and @dckc also is [suggesting the same amount for the budget](https://github.com/rchain/bounties/issues/823#issuecomment-404297930)

cc: @deannald @dckc

**Comment by dckc** [comment]
That suggestion of mine was for a particular meeting which seemed to have little value because no progress was made on any of the actions recorded in the meeting. I'm not aware that it applies to this meeting.

**Comment by pmoorman** [comment]
My thinking was a significant budget for the first month, considering all the hustle and bustle to get this routine started (hence the $3000 vote), and then a fixed "per meeting" budget moving forward (much like you suggest).

**I think RAAM should be considered as equal to the RAM meeting, just 12 hours apart.**

**Comment by deannald** [comment]
I agree with @dckc and @pmoorman in that a fixed per meeting budget moving forward makes sense and should be commensurate with the amount of effort put into the meetings.

**Comment by dckc** [comment]
If you agree with me, then you're not arguing for a fixed budget :)

From what I have seen in RAM meetings, the quality of meetings varies significantly from meeting to meeting. I'm OK with budget commensurate with effort to the extent that putting effort into a meeting usually produces quality results. But it's the results that I'm really looking for.

I don't have much direct experience with the Asia meetings, but from what I hear, $3000 for the first month was earned.

**Comment by zsluedem** [comment]
@kitblake @pmoorman 
I suggest 1500$ for the work we did this month. 
50% for @zsluedem
20% for @cool007zqw 
20% for @kitblake 
10% for @pmoorman
